api/server.js

```diff
@@ -0,0 +1,28 @@
+// require('dotenv').config();
+const express = require('express');
+const server = express();
+server.use(express.json());
+
+// CONFIG: server settings
+const serverPort = process.env.PORT || 7100; // server port
+const serverName = `Sprint-Challenge--Testing`; // Name of server to display at "/" endpoint 
+const projectPullRequest = `https://github.com/michaelagard/Sprint-Challenge--Testing`;
+let gameArray = [];
+// ENDPOINTS
+server.get('/', (req, res) => { // sanity check root endpoint
+  res.send(`${serverName} running on port ${serverPort}<br>More information: <a href="${projectPullRequest}">{serverName}</a>`);
+});
+server.get('/api/games', (req, res) => { // sanity check root endpoint
+  res.status(200).json(gameArray);
+});
+server.post('/api/games/', (req, res) => {
+  let { title, genre, releaseYear } = req.body;
+  if (title && genre) {
+    gameArray.push(req.body)
```

Comment:
In a real world scenario, you wouldn't want to add the entire request body to your database since there could be malicious data in it. Even though it's not as important here, it's still best to write code as you would in a professional environment.